### ISSUE TITLE
Verify backdrop orgs on GitHub

### ISSUE THREAD

**Issue by klonos** [backdrop-ops/backdropcms.org#511]
We should verify b.org as a domain for:

- https://github.com/backdrop-contrib
- https://github.com/backdrop
- https://github.com/backdrop-ops

<img width="1013" alt="screen shot 2019-03-07 at 6 38 58 am" src="https://user-images.githubusercontent.com/2423362/53908706-13a67100-40a4-11e9-98c8-6bdfb9c6ff17.png">

**Here's how**: https://help.github.com/en/articles/verifying-your-organizations-domain

Basically:

- add `backdropcms.org` in the "Verified domains" section on the settings for each of these GitHub orgs (I personally only have access to `backdrop-contrib`).
- add TXT records as per instructions provided by GitHub

**Here's why**:

![org_profile_verified](https://user-images.githubusercontent.com/2423362/53909444-cfb46b80-40a5-11e9-8f5a-bba019d365a8.png)

**Comment by jenlampton** [comment]
yes! I have set them up and emailed the txt records to @quicksketch for set-up.

**Comment by jenlampton** [comment]
this is done :)
